src/main/java/io/dropwizard/actors/actor/Actor.java

```diff
@@ -61,9 +61,9 @@ protected Actor(
     abstract protected boolean handle(Message message) throws Exception;
 
     protected boolean isExceptionIgnorable(Throwable t) {
-        return droppedExceptionTypes
-                .stream()
-                .anyMatch(exceptionType -> ClassUtils.isAssignable(t.getClass(), exceptionType));
+        return droppedExceptionTypes != null && !droppedExceptionTypes.isEmpty()
```

Comment:
Set this to empty set in constructor if it is sent as null. Much more efficient. Do not do this.